src/app/page.tsx

```diff
@@ -156,9 +156,7 @@ const Home = async ({ searchParams }: Props) => {
             <TwitterModule tweets={tweets} />
           </section>
         )}
-        <section className="w-full">
-          <Trending />
-        </section>
+        <Trending />
```

Comment:
why drop the sections? @ximxim

Reply:
If I left the section, there was additional space when the component was gone. I moved the styled to the compone so it hides seamlessly and retains it's previous look when it is back.